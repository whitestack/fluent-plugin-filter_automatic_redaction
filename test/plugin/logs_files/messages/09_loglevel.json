[
  {
    "input_message": "Lorem Ipsum [INFO] Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum [WARN] Lorem Ipsum"
  }
]