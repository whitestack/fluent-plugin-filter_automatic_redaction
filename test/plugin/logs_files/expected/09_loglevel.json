[
  {
    "input_message": "Lorem Ipsum [INFO] Lorem Ipsum",
    "output_message": "Lorem Ipsum [<LOGLEVEL>] Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum [WARN] Lorem Ipsum",
    "output_message": "Lorem Ipsum [<LOGLEVEL>] Lorem Ipsum"
  }
]